source/system/system.go

```diff
@@ -153,6 +163,16 @@ func splitVersion(version string) map[string]string {
 	return components
 }
 
+// Read /sys/devices/virtual/dmi/id/sys_vendor
+func getVendor() (string, error) {
```

Comment:
Just a suggestion to make this generic for exposing other attributes

```suggestion
func getDmiIdAttribute(name string) (string, error) {
```

Reply:
Thanks, makes sense too, updated.